hips/draw/simple.py

```diff
@@ -65,12 +29,52 @@ class SimpleTilePainter:
         HiPS survey properties
     tile : `HipsTile`
```

Comment:
The `SimpleTilePainter` no longer takes a `HipsTile` in `__init__`.

Also the description above is no longer correct, it still says "paint a single tile"

Update it's docstring!